pkg/geo/geo.go

```diff
@@ -831,6 +831,7 @@ func validateGeomT(t geom.T) error {
 			}
 		}
 	case *geom.GeometryCollection:
+		// TODO(ayang): verify that the geometries all have the same Layout
```

Comment:
Oh I left this TODO since you said in an earlier comment we should check it again just in case https://github.com/cockroachdb/cockroach/pull/60515#discussion_r574931509. I can remove it if it's no longer needed.

Reply:
Isn't it already in `validateGeomT`?

Reply:
we don't validate layouts in validateGeomT (yet)